docs/ballerina-by-example/examples/record-i-o/record-i-o.bal

```diff
@@ -13,21 +13,47 @@ function getFileRecordChannel (string filePath, string permission, string encodi
     return delimitedRecordChannel;
 }
 
+@Description{value:"This function will read next record from the channel"}
+function readNext(io:DelimitedRecordChannel channel)(string []){
+    io:IOError err;
+    string[] records;
+    records, err = channel.nextTextRecord();
```

Comment:
Shall we use var here?

Reply:
As discussed with @anupama-pathirage we will be leaving the samples as it is to preserve readability instead of using var